lib/node_modules/@stdlib/_tools/scaffold/math-iter-unary/scripts/scaffold.sh

```diff
@@ -37,23 +37,23 @@
 ## USER-DEFINED VARIABLES ##
 
 # Define the main export alias:
-alias=${ALIAS:-'iterTest'}
+alias='iter'$(echo ${ALIAS:-'Test'} | awk '{print toupper(substr($0, 1, 1)) tolower(substr($0, 2))}')
```

Comment:
I think my preference would be to move the alias logic to JavaScript. While it is common for us to do `iterXxxxx`, it is not always the case. So I'd revert to

```
${ALIAS:-'iterTest'}
```

and then in the runner.js file do

```
envs.push( 'ALIAS=iter'+capitalize( v ) );
```

or something similar.

Reply:
Done, thanks!